content/en/docs/tasks/observability/gateways/index.md

```diff
@@ -84,7 +84,7 @@ the [Secret Discovery Service](https://www.envoyproxy.io/docs/envoy/latest/confi
 
     {{< text bash >}}
     $ cat <<EOF | kubectl apply -f -
-    apiVersion: certmanager.k8s.io/v1alpha1
+    apiVersion: cert-manager.io/v1alpha2
```

Comment:
There is one more change needed in the `Cleanup` section

Reply:
done